cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/PushUtils.java

```diff
@@ -75,8 +76,10 @@
      * @param packageName   path of the package folder to be pushed
      * @param sourceRoot    path to the directory containing source files and packages
      * @param installToRepo if it should be pushed to central or home
+     * @param pushToRepo true if confirmation is not required before push, else false
      */
-    public static void pushPackages(String packageName, String sourceRoot, String installToRepo) {
+    public static void pushPackages(String packageName, String sourceRoot, String installToRepo,
+                                    boolean pushToRepo) {
```

Comment:
Isn't it pushToCentral ?

Reply:
The same method is used both by the push command and the init command. In the case of push its "pushToCentral" and in the case of install "pushToHomeRepo". To be generic in both the cases I added it as "pushToRepo"